models/object_tracking_vittrack/demo.java

```diff
@@ -0,0 +1,206 @@
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.UnixStyleUsageFormatter;
+import org.bytedeco.javacpp.BytePointer;
+import org.bytedeco.opencv.global.opencv_dnn;
+import org.bytedeco.opencv.opencv_core.*;
```

Comment:
> org.bytedeco.opencv

This doesn't look as original OpenCV Java API.

/cc @asmorkalov

Reply:
Yes, it's 3rdparty package. We do not distribute pre-built Java package with Maven Central.

Reply:
> We do not distribute pre-built Java package with Maven Central

So there is nothing for Java like Python does (pip install then use out-of-the-box)?